tests/sentry/api/test_event_search.py

```diff
@@ -156,6 +156,63 @@ def test_simple(self):
             ),
         ]
 
+    def test_simple_in(self):
+        assert parse_search_query("user.email:[test@test.com] test:[hello]") == [
+            SearchFilter(
+                key=SearchKey(name="user.email"),
+                operator="IN",
+                value=SearchValue(raw_value=["test@test.com"]),
+            ),
+            SearchFilter(
+                key=SearchKey(name="test"),
+                operator="IN",
+                value=SearchValue(raw_value=["hello"]),
+            ),
+        ]
+        assert parse_search_query(
+            "user.email:[test@test.com,test2@test.com,test3@test.com] test:[hello]"
```

Comment:
Do we need to have special logic for handling IN on the array fields? stack.filename and error.type come to mind.

Reply:
We might, I assume the handling of array fields happens later on in `convert_search_filter_to_snuba_query` or some other related step?

Not sure how we would handle this on array fields actually, I assume it'd be `if any value from test_array exists in array_col`.

Reply:
My original thought when suggesting using this `[]` syntax was that we could just turn
`array_field:[1,2,3]` to mean `[array_field, '=', [1,2,3]]`
especially since we don't currently have a way to do `equals` for array fields